pkg/bot/mattermost.go

```diff
@@ -185,17 +235,17 @@ func (mm mattermostMessage) sendMessage() {
 		}
 		post.FileIds = []string{res.FileInfos[0].Id}
 	} else {
-		post.Message = formatCodeBlock(mm.Response)
+		post.Message = format2.CodeBlock(mm.Response)
 	}
 
-	// Create a post in the Channel
+	// Create a post in the ChannelName
```

Comment:
```suggestion
	// Create a post in the ChannelID
```
or just remove it. Personally, I don't see a value in this comment.